extensions/amp-base-carousel/1.0/amp-base-carousel.js

```diff
@@ -110,7 +110,7 @@ AmpBaseCarousel['props'] = {
     attrs: ['snap'],
     parseAttrs: (element) => parseStrBoolAttr(element, 'snap', true),
   },
-  'visibleCount': {attr: 'visible-count', type: 'number'},
+  'visibleCount': {attr: 'visible-count', type: 'number', media: true},
```

Comment:
Added `{media: true}` to more props. However, I'm not yet ready to make it default because the parse rules are relatively brittle. E.g. spaces, commas, and parens cannot be used for this format to work correctly and this might not work for all props.

Another problem: `snap` and `loop` are currently not "media-fied" for related but different reasons.